public/stylesheets/main.css

```diff
@@ -12,4 +12,14 @@
 
 #delete-form {
     display: inline;
+}
+
+.form-heading {
+    text-align: center;
+    margin-top: 30px;
+}
+
+.form {
+    width: 30%;
+    margin: 30px auto;
 }
```

Comment:
Having a single blank line at the end of the file is a common practice.

Reply:
Thank you! I already changed it, but I don't want to commit it now, just for this change.